tox/session.py

```diff
@@ -179,7 +179,7 @@ def popen(self, args, cwd=None, env=None, redirect=True, returnout=False, ignore
                     out, err = popen.communicate()
             except KeyboardInterrupt:
                 self.report.keyboard_interrupt()
-                popen.wait()
+                popen.kill()
```

Comment:
Not really related to your change, but I think this is reasonable anyway - why should I wait for the process, when I  try to interrupt the whole run.

Reply:
shouldnt this send a sigint first instead of sigkill/sigterm?

Reply:
I used kill here considering the case where the sigint handler even hangs. Would it be a problem?
Following is an example:
```python
def hang(*args):
    while True:
        pass
signal(signal.SIGINT, hang)
os.kill(0, signal.SIGINT)
```

Reply:
@Suckzoo thanks for the note - i believe later on we might want to give it some time to react to a sigint passing on the sigterm later on to ensure shutdown after a timeout

Reply:
@RonnyPfannschmidt I see. I'll look over it by tomorrow... hopefully.
btw, I think it will be good to pass time limit value as an argument. In which file should I look?